sealing/sealing.go

```diff
@@ -47,11 +53,25 @@ func NewSealing(api node.Interface, sb sectorbuilder.Interface, ds datastore.Bat
 }
 
 func NewSealingWithOnSectorUpdated(api node.Interface, sb sectorbuilder.Interface, ds datastore.Batching, maddr address.Address, onSectorUpdated func(abi.SectorNumber, SectorState)) *Sealing {
+	// TODO: This value represents the quantity of epochs into the future (from
+	// the chain head-epoch) at which point we expect the pieces in a newly
+	// created self-deal to have been sealed into a proven sector, which will
+	// be a function of sector size (among other things). For now, we pick a
+	// number sufficiently large as to ensure that a miner will have sufficient
+	// time to replicate and prove a 16MiB self deal-packed sector.
+	commitDelay := abi.ChainEpoch(2 * 60 * 24) // ~1 day into the future, assuming 30s block time
```

Comment:
Curious if this concept is represented at all in the storage-market.

Reply:
Great question. The [answer is](https://filecoinproject.slack.com/archives/CPLLE0ZQX/p1582233213127800): no.